lib/Builder.js

```diff
@@ -91,9 +91,10 @@ class Builder {
             }
         } catch (ex) {
             Util.logger.error('mcdev.buildDefinition:' + ex.message);
-            Util.logger.debug(ex.stack);
+            const stack = new Error().stack;
```

Comment:
how does this even work? wont stack of a new Error always be null? or does it kind of know where it is?

Reply:
nope it wont.
quoting the PR text:
"In certain cases we wanted a stack trace in our logs our CLI output. this was so far using Error.stack but as depicted in the issue, that's weirdly not always the best way. Instead, creating a NEW error and using that error's stack tends to do the trick of showing the full stack."

Reply:
for more details please check the bug ticket where I supplied screenshots: #222